os/other-settings.md

```diff
@@ -33,10 +33,31 @@ coreos:
       command: restart
 ```
 
+### Loading Kernel Modules with Options
+
+This example will load `dummy` module with `numdummies=5` option:
```

Comment:
This example cloud-config excerpt loads the `dummy` network interface module with configuration specifying the number of interfaces the module should create when loaded (`numdummies=5`):